pkgs/applications/virtualization/xen/generic.nix

```diff
@@ -246,8 +246,19 @@ stdenv.mkDerivation (rec {
     platforms = [ "x86_64-linux" ];
     maintainers = with lib.maintainers; [ eelco oxij ];
     license = lib.licenses.gpl2;
+    knownVulnerabilities = [
+      # https://www.openwall.com/lists/oss-security/2023/03/21/1
+      # Affects 3.2 (at *least*) - 4.17
+      "CVE-2022-42332"
+      # https://www.openwall.com/lists/oss-security/2023/03/21/2
+      # Affects 4.11 - 4.17
+      "CVE-2022-42333"
+      "CVE-2022-42334"
+      # https://www.openwall.com/lists/oss-security/2023/03/21/3
+      # Affects 4.15 - 4.17
+      "CVE-2022-42331"
     # https://xenbits.xen.org/docs/unstable/support-matrix.html
-    knownVulnerabilities = lib.optionals (lib.versionOlder version "4.13") [
+    ] ++ lib.optionals (lib.versionOlder version "4.16") [
```

Comment:
I think we are talking about the package being unmaintained with known security problems. The version itself is not EOL.

Reply:
Yeah I was going to mention the support matrix but I misread it.